suite-native/receive/src/hooks/receiveSelectors.ts

```diff
@@ -0,0 +1,9 @@
+import { pipe, A } from '@mobily/ts-belt';
+
+import { DeviceRootState, selectDeviceButtonRequestsCodes } from '@suite-common/wallet-core';
+
+export const hasReceiveAddressButtonRequest = (state: DeviceRootState) =>
+    pipe(
+        selectDeviceButtonRequestsCodes(state),
+        A.some(code => code === 'ButtonRequest_Address'),
```

Comment:
Don't we have some const available somewhere instead of text?

Reply:
Unfortunetaly no, that's why I created a selector for this. Luckily though, it's typed well so any typo will be caught with TS error